app/controllers/user_onboarding_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UserOnboardingController < ApplicationController
   skip_before_action :ensure_onboarded!
+  skip_before_action :store_user_location!
```

Comment:
If we add the `user_signed_in?` bit as per my comment above, we won't need this here.